workflows/download_device_design_data_as_landxml/download_device_design_data_as_landxml.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/python
+import argparse
+import logging
+import os
+import sys
+import requests
+import json
+from tqdm import tqdm
+
+sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__)), "..", "..", "components", "tokens"))
+sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__)), "..", "..", "components", "utils"))
+sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__)), "..", "..", "components", "files", "file_download"))
+sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__)), "..", "..", "components", "files", "file_list"))
+sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__)), "..", "..", "components", "files", "file_features"))
+sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__)), "..", "..", "components", "metadata", "metadata_list"))
+
+from get_token import *
+from utils import *
+from args import *
+from file_download import *
+from file_list import *
+from file_features import *
+from metadata_list import *
+
+session = requests.Session()
+
+def particular_from_design_type(a_design_type):
+    if(a_design_type == "Lines"):
+        return "LN3"
+    elif(a_design_type == "Planes"):
+        return "PL3"
+    elif(a_design_type == "Points"):
+        return "PT3"
+    elif(a_design_type == "Roads"):
+        return "RD3"
+    else:
```

Comment:
missing particular "TN3" for design type "Surfaces"

Reply:
Thank you.